src/pages/InvalidUser/index.js

```diff
@@ -0,0 +1,26 @@
+import LogoutButton from 'components/LogoutButton';
+import useAuth from 'hooks/useAuth';
+
+import './styles.css';
+
+const InvalidUser = () => {
+  const { user } = useAuth();
+
+  const { email, name: completeName } = user || {};
+
+  const name = completeName?.split(' ')[0];
+
+  return (
+    <div className="invalid-user">
+      <p className="invalid-user-message">
+        {`Lo siento ${name} pero tu email "${email}" no pertenece a Rootstrap.`}
```

Comment:
What do you think about implementing a library for internalization? Or at least a file that has all the verbiage together

Reply:
Yes, I have that pending. I haven't done it yet 😅  But I know I should change it. In another PR I will localize all texts (they are a lot 😅 )